### PR TITLE
MCA Reborn

### LINKED ISSUE #59: Request compat for: Minecraft Comes Alive Reborn (1.16)
I would like to see integrations added for the mod mentioned in the title.  
I confirm it is available for the latest Minecraft version at the time of writing.

## About the mod
Link to the mod page (e.g. on CurseForge):

Link to mod source/API:

### PR DESCRIPTION
**Pull request by maxanier**
Update old MCA 1.12 integration for MCA Reborn on 1.16.
Closes #59

### CODE REVIEW

**Comment by JasonPra** [comment]
I am happy to notice your already on this :)

**Comment by Cheaterpaul** [comment]
Since there are no different entity ids for different villager and genes are use instead the face overlay determines the texture based on the face gene

File: gradle/forge.gradle [hunk 1]
```diff
@@ -162,21 +159,24 @@ dependencies {
     runtimeOnly fg.deobf("slimeknights.mantle:Mantle:1.16.5-"+project.mantle_version)
 
     // Survive - https://www.curseforge.com/minecraft/mc-mods/survive
-    implementation fg.deobf("curse.maven:survive-367337:"+project.survive_fileid)
+    compileOnly fg.deobf("curse.maven:survive-367337:"+project.survive_fileid)
```
Comment (Cheaterpaul) [inline gradle/forge.gradle:162]: running with survive throws a mixin inject error for me. With old and new version

File: src/main/java/de/teamlapen/vampirism_integrations/mca/EntityAngryVillagerMCA.java [hunk 2]
```diff
@@ -29,64 +39,59 @@
  */
 public class EntityAngryVillagerMCA extends EntityVillagerVampirismMCA implements IAggressiveVillager {
 
-    private ItemStack pitchforkStack;
-    private AxisAlignedBB area;
-
-
-    public EntityAngryVillagerMCA(World world) {
-        super(world);
-        pitchforkStack = new ItemStack(MCACompatREFERENCE.pitchfork);
```
Comment (Cheaterpaul) [inline src/main/java/de/teamlapen/vampirism_integrations/mca/EntityAngryVillagerMCA.java:38]: Why shouldn't they have pitchforks? Are they unable to use items or was it not finished yet?

**Review by Cheaterpaul — COMMENTED** [review]
TODO:
- [ ] Angry villages are missing faces
- [ ] re-add runtime survive
- [ ] maybe check required mca version

File: gradle.properties [hunk 3]
```diff
@@ -4,11 +4,11 @@ main_version=1
 major_version=7
 minor_version=0
 minecraft_version=1.16.5
-forge_version=36.2.4
+forge_version=36.2.26
 mcp_mappings=1.16.5
 mcp_channel=official
 vampirism_mcversion=1.16.5
-vampirism_version=1.8.3
+vampirism_version=1.8.9
```
Comment (Cheaterpaul) [inline gradle.properties:11]: Since the TeamLapen/Vampirism@3fa8dc89 is not published yet, i used a local version

**Pull request closed without merging**